apps/redi-connect/src/pages/app/me/Me.tsx

```diff
@@ -1,6 +1,7 @@
-// import { usePatchMyProfileMutation } from '@talent-connect/data-access'
+import { Tooltip } from '@mui/material'
 import {
   Button,
+  Checkbox,
```

Comment:
Shouldn't we rather be using `Checkbox` from `@talent-connect/shared-atomic-design-components`?

Reply:
We are using the `Checkbox` from `@talent-connect/shared-atomic-design-components` here 🤔

Reply:
Huh, brainfart 😵‍💫 OK, then good to go ✅